sdk/src/types/block/output/feature/block_issuer.rs

```diff
@@ -63,6 +75,17 @@ impl Ed25519BlockIssuerKey {
     pub fn try_from_bytes(bytes: [u8; Self::LENGTH]) -> Result<Self, Error> {
         Ok(Self(ed25519::PublicKey::try_from_bytes(bytes)?))
     }
+
+    pub(crate) fn null() -> Self {
+        // Unwrap: we provide a valid byte array
```

Comment:
Are we sure full 0 is actually a valid public key ?

Reply:
Works
```Rust
    let public_key = prefix_hex::decode("0x0000000000000000000000000000000000000000000000000000000000000000")?;
    let public_key = PublicKey::try_from_bytes(public_key)?;
    println!("{:?}", public_key);
```
```
PublicKey(VerificationKey { A_bytes: VerificationKeyBytes("0000000000000000000000000000000000000000000000000000000000000000"), minus_A: EdwardsPoint{
        X: FieldElement51([533094393274173, 2016890930128738, 18285341111199, 134597186663265, 1486323764102114]),
        Y: FieldElement51([0, 0, 0, 0, 0]),
        Z: FieldElement51([1, 0, 0, 0, 0]),
        T: FieldElement51([2251799813685229, 2251799813685247, 2251799813685247, 2251799813685247, 2251799813685247])
} })
```

Reply:
Thanks